src/ActiveLogin.Authentication.BankId.AspNetCore/Events/LoggerBankIdEventListner.cs

```diff
@@ -0,0 +1,82 @@
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Events
+{
+    public class LoggerBankIdEventListner : TypedBankIdEventListner
+    {
+        private const string MissingPersonalIdentityNumber = "-";
+        private readonly ILogger<LoggerBankIdEventListner> _logger;
+
+        public LoggerBankIdEventListner(ILogger<LoggerBankIdEventListner> logger)
```

Comment:
I don't think there is much to do about it, but I realize that it might be a breaking change. Now we will log from `ILogger<LoggerBankIdEventListner>` while it was different `ILogger<T>` before. I don't think we should do anything about that.

Reply:
Left this unchanged.